drivers/sensor/lsm6dsr/lsm6dsr.h

```diff
@@ -0,0 +1,98 @@
+/* ST Microelectronics LSM6DSO 6-axis IMU sensor driver
+ *
+ * Copyright (c) 2019 STMicroelectronics
```

Comment:
Do you work for ST? otherwise the copyright should be you?

Reply:
This file is cut&pasted from other LSM6xxx drivers.  In fact, the comment at the top still says `LSM6DSO` and not `LSM6DSR`.  So it wouldn't really be true to say it was an entirely new file without the existing copyright.

The design of the ST HAL and drivers is such that the entire driver is cut & pasted to a new driver, which uses a completely different HAL, if even one single register has just one bit different.